### PR TITLE
Add the behaviorbot Welcome bot

### PR DESCRIPTION
**Pull request by nsryan2**
This PR adds a file congif.yml to the .github folder. This is the bot that @kkiesling found on YT's repository, but the messages have been adapted to better fit PyNE. I'm not completely confident that it works, so I made this a draft.

### CODE REVIEW

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hi, and welcome to PyNE!  Thanks for opening your first issue.  We recommend that you 
+  include information such as the version of Pyne you're working with, the platform you
```
Comment (kkiesling) [inline .github/config.yml]:
```suggestion
  include information such as the version of PyNE you're working with (eg, develop branch or a specific version), the platform you
```

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  are operating on, the expected behavior, and the actual behavior you are bringing our 
+  attention to. The more deatil you provide, the better others in this community will 
+  be able to help you.
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Hi!  Welcome, and thanks for opening this pull request.  We have some guidelines for
```
Comment (kkiesling) [inline .github/config.yml]: Should we link to guidelines? Do we have specific guidelines other than it must pass CI?

**Review by kkiesling — COMMENTED** [review]
this makes me so happy!

File: .github/config.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hi, and welcome to PyNE!  Thanks for opening your first issue.  We recommend that you
```
Comment (gonuke) [inline .github/config.yml]:
```suggestion
  Hi, and welcome to PyNE!  :wave: Thanks for opening your first issue.  We recommend that you 
```
It's always good to add emojis :smile:

**Review by gonuke — CHANGES_REQUESTED** [review]
I've enabled the bot to use this once it's merged.

File: .github/config.yml (hunk 2 above)
Comment (nsryan2) [reply .github/config.yml]: We do not have a specific set of guidelines, I forgot to take that out. I can add a template for PRs, and change the line above to reflect that if we'd like more structure.

**Comment by nsryan2** [comment]
> I've enabled the bot to use this once it's merged.

Thank you! I wasn't sure how to do that part.

**Comment by bam241** [comment]
this need to be rebased.

**Comment by bam241** [comment]
you'll need to update the changelog file

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+## Types of changes
```
Comment (kkiesling) [inline .github/pull_request_template.md]: does this file appear somewhere with the bot? I don't think we need this. I thought we had contributing guidelines online that we could reference.

Comment (nsryan2) [reply .github/pull_request_template.md]: I can get rid of this template for sure, so far as I know we don't have specifics on what should go into a PR anywhere on the site. The [building the website](http://pyne.io/devsguide/website.html#procedure-for-modifying-the-website) page has a procedure for modifying the website, but it doesn't have details for PRs

**Comment by gonuke** [comment]
Thanks @nsryan2  - let's see how it works...

**Review by gonuke — APPROVED** [review]

**Pull request merged**